pipelines/nuget-package/default-ci-pipeline.yaml

```diff
@@ -0,0 +1,66 @@
+# [default] pipeline for nuget package project, use for [master] branch
+# suitable for simplified gitflow =>
+# branch under /feature folder: no package is created by default by CI, if needed, specify pack variable => ALPHA version will be created
```

Comment:
IMHO this documentation should be in dedicated *.MD file that could be referenced from yaml pipeline file.

Reply:
docs added